test/factories/factories.rb

```diff
@@ -62,6 +65,10 @@
     "Dev#{n}"
   end
 
+  sequence :last_name do |n|
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.